packages/flutter_tools/lib/src/build_runner/devfs_web.dart

```diff
@@ -114,6 +114,41 @@ class WebAssetServer implements AssetReader {
         shelf.serveRequests(httpServer, releaseAssetServer.handle);
         return server;
       }
+      // Return the set of all active modules. This is populated by the
+      // frontend_server update logic.
+      Future<Map<String, String>> moduleProvider(String path) async {
+        return modules;
+      }
+      // Return a version string for all active modules. This is populated
+      // along with the `moduleProvider` update logic.
+      Future<Map<String, String>> digestProvider(String path) async {
+        return digests;
+      }
+      // Return the module name for a given server path. These are the names
+      // used by the browser to request JavaScript files.
+      String moduleForServerPath(String path) {
+        if (serverPath.endsWith('.lib.js')) {
+          serverPath = serverPath.startsWith('/')
+            ? serverPath.substring(1)
+            : serverPath;
+          return serverPath.replaceAll('.lib.js', '');
+        }
+        return null;
+      }
+      // Return the server path for modules. These are the JavaScript file names
+      // output by the frontend_server.
+      String serverPathForModule(String module) {
+        return '$module.lib.js';
+      }
+      // Return the server path for modules that have an org-dartlang-app
```

Comment:
This isn't limited to just modules. It is used for any `org-dartlang-app` resource which includes Dart files.

Reply:
Fixed